frontend/lib/src/components/widgets/ChatInput/ChatInput.tsx

```diff
@@ -227,10 +227,12 @@ function ChatInput({
                   : "auto",
                 maxHeight: maxHeight ? `${maxHeight}px` : "none",
                 // Baseweb requires long-hand props, short-hand leads to weird bugs & warnings.
-                paddingRight: "3rem",
                 paddingLeft: theme.spacing.sm,
                 paddingBottom: theme.spacing.sm,
                 paddingTop: theme.spacing.sm,
+                // Calculate the right padding to account for the send icon (iconSizes.xl + 2 * spacing.sm)
```

Comment:
This comment mainly repeats the formula itself. `iconSizes.xl` -> send icon? what is `* spacing.sm`? Even if it is some arbitrary factor, this should be stated. Something like
`* spacing.sm` is an arbitrary factor to get the same size we have used historically (`3rem`) but now derived from our theme.

Reply:
Updated the comment a bit. I think the historical aspect of 3rem isn't that important to include. What we want here is some additional margin between the input text and the send icon, and `spacing.sm` works fine.

Reply:
Yeah that makes sense! I think adding the intention "additional margin between the input text and the send icon, and spacing.sm works fine" would be a good info to add